regolith/helpers/Google_Calendar_API/google_calendar_api.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import print_function
+import datetime
+import os
+import pathlib
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+
+
+def add_to_calendar(event):
+    """ Takes a newly created presentation, and adds it to the user's google calendar """
```

Comment:
This docstring needs to be up to standard.

Also, this function looks as if it takes an _event_ and adds it to the calendar, not a presentation.  That is better tbh, because we may want to use this function in other helpers etc. and add other things to the calendar.